.github/workflows/amd_ci.yml

```
@@ -116,4 +115,4 @@ jobs:
         source "${{ runner.tool_cache }}/${{ env.VENV_DIR }}/bin/activate"
         cd testing/python/amd
         unset PYTHONPATH
-        python -m pytest -v test_tilelang_test_amd.py
\ No newline at end of file
+        python -m pytest -v test_tilelang_test_amd.py --durations=0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)